frontend/packages/operator-lifecycle-manager/integration-tests-cypress/views/operator.ts

```diff
@@ -0,0 +1,144 @@
+import { projectDropdown } from '@console/cypress-integration-tests/views/common';
+import { detailsPage } from '@console/cypress-integration-tests/views/details-page';
+import { listPage } from '@console/cypress-integration-tests/views/list-page';
+import { modal } from '@console/cypress-integration-tests/views/modal';
+import { nav } from '@console/cypress-integration-tests/views/nav';
+
+export const operator = {
+  install: (
+    operatorName: string,
+    operatorHubCardID: string,
+    installToNamespace: string,
+    installToSingleNamespace: boolean = false,
```

Comment:
This argument isn't really necessary. We can just take 3 args, and if the `installToNamespace` arg is undefined, we assume a global install and use the 'openshift-operators' namespace.

Reply:
I did have it that way, but openned it up to be more consistant between single and global installs, and provide maximum flexibility -wasn't sure if it always `openshit-operators` for every Operator (I know it is for RH Data Grid we're using, and probably for others, but don't know if that will always 100% of the time be the case).

Reply:
In current OLM, it will always be 'openshift-operators'. That's a system namespace that's specifically there for globally installed operators.